app/main/posts/modify/post-editor.html

```diff
@@ -33,15 +33,15 @@ <h1 class="mode-context-title">{{post.form.name}}</h1>
 
           <div class="form-sheet">
               <div class="post-band" style="background-color: {{form.color}}"></div>
-              <div class="form-field title form-field-adapt init"
+              <div class="form-field title init required"
```

Comment:
They seem to be issues caused by the newer PL, I noticed them when testing the posts after import. I can pull them out into their own PR though.